src/fasttext_lite/core.py

```diff
@@ -37,8 +37,8 @@ def convert_path(path: StrOrPath) -> Path:
 
 class BaseFastTextClassifier(ABC, BaseEstimator, ClassifierMixin):
     @abstractmethod
-    def fit(self):
-        pass
+    def fit(self, X, y) -> "BaseFastTextClassifier":
+        return self
```

Comment:
Per further reading, matching the sklearn convention of fit returning self, to enable method chaining like `model.fit().predict()